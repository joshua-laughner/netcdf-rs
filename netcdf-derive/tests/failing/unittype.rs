#[derive(netcdf_derive::NcType)]
#[repr(C)]
struct B;

fn main() {}
