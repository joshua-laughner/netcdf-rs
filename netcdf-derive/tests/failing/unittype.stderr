error: Can not derive NcType for unit struct
 --> tests/failing/unittype.rs:2:1
  |
2 | / #[repr(C)]
3 | | struct B;
  | |_________^
